session.go

```diff
@@ -221,11 +225,68 @@ func (s *goCloakSession) AddAuthTokenToRequest(client *resty.Client, request *re
 		tokenType = token.TokenType
 	}
 
-	request.Header.Set("Authorization", tokenType+" "+token.AccessToken)
+	request.Header.Set(headerAuthorization, tokenType+" "+token.AccessToken)
 
 	return nil
 }
 
+func (s *goCloakSession) GRPCUnaryAuthenticate() grpc.UnaryClientInterceptor {
+	return func(
+		ctx context.Context,
+		method string,
+		req, reply interface{},
```

Comment:
Not a mistake or anything, but isn't this usually called "response", not "reply"? Or is this different in GRPC as opposed to REST?

Reply:
took this straight from the documentation of the [google.golang.org/grpc](https://pkg.go.dev/google.golang.org/grpc) package 👍